buildscripts/manylinux2014/configure_conda.sh

```diff
@@ -0,0 +1,16 @@
+# Setup miniconda environment that is compatible with manylinux2014 docker image
```

Comment:
add #!/bin/bash as [[ is bashism.

Reply:
done in 2f0a24af7e3b975819346638a146046faa82784e